packages/core-ethereum/src/ethereum.ts

```diff
@@ -154,12 +156,14 @@ export async function createChainWrapper(
       // compute tx hash and save to initiated tx list in tx manager
       const initiatedHash = utils.keccak256(signedTx)
       transactions.addToQueuing(initiatedHash, { nonce, gasPrice }, essentialTxPayload)
-
+      // with let indexer to listen to the tx
+      removeListener = handleTxListener(initiatedHash)
```

Comment:
If anything above throws then, `removeListener` is undefined

Reply:
Close this discussion due to changes in the implementation